diff --git a/src/Makefile b/src/Makefile
index 4bea83b4..a0233be2 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -43,11 +43,11 @@ export CXX ?= g++
 export AS := yasm
 export RANLIB ?= ranlib
 
-export CFLAGS := -Wall
-export CXXFLAGS := -Wall -Wno-unused-parameter
+export CFLAGS := ${CFLAGS} -Wall
+export CXXFLAGS := ${CXXFLAGS} -Wall -Wno-unused-parameter
 C_CXX_FLAGS := -MMD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGE_FILES -I$(BASE_DIR) -I$(BASE_DIR)/Crypto
 export ASFLAGS := -D __GNUC__ -D __YASM__
-export LFLAGS :=
+export LFLAGS := ${LDFLAGS}
 
 export PKG_CONFIG ?= pkg-config
 export PKG_CONFIG_PATH ?= /usr/local/lib/pkgconfig
